test/fixtures/hello-cloud/crd.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Adding this to hello-cloud (which strictly speaking is correct--I usually want all supported resources to be in this set) makes me want to build a real demo directory like we've been asked to, so that we don't _encourage_ this gem to be used for globals.